library/std/src/sys/teeos/time.rs

```diff
@@ -0,0 +1,348 @@
+//! copy from unix 100%, version 1.65
```

Comment:
If this is an exact copy of unix/time.rs, it might be better to just import that file directly from mod.rs:

```rust
#[path = "../unix/time.rs"]
pub mod time;
```

Reply:
(And similar for `thread_local_key.rs` and perhaps more.)

Reply:
some code under unix is not allowed to be imported

Reply:
time.rs can not be imported, thread_local_key.rs is optimized.

Reply:
It seems the issue is due to `pub(in crate::sys::unix)` in the unix implementation. Maybe it would be better to just change that to `pub(crate)` to allow the code to be reused (timespec is very widespread outside "unix" targets).

Reply:
Yes, please just change the Unix code to use `pub(crate)` to avoid the code duplication.